docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -499,7 +496,7 @@ def compute_eti(dataset, eti_prob):
 
 fig, axes = plt.subplots(figsize=(6, 5))
 az.plot_hdi(
-    idata.prior_predictive["negbinom_rv_dim_0"] + gen_int.size(),
+    idata.prior_predictive["negbinom_rv_dim_0"],
```

Comment:
I think these should have `+ n_initialization_points`. This is to make the time points across the plots agree. Alternatively, you could start with a negative time on the other plots.

Reply:
I removed the shift in x-axis so the plot with forecast has same time points as the posterior predictive plot. Will change the `latent_infection` plots to start from `- n_initialization_points`

Reply:
Actually, it's easier to have axis start from `n_initialization_points` than `-n_initialization_points` when plotting individual drwas using the `plot_posterior` method.

Reply:
I removed one of the plots with posterior draws of `latent_infections` in favor of uniformizing x-axis across all the figures but I can bring it back if we feel strongly about it.